modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/RuleEditor.es.js

```diff
@@ -31,6 +31,34 @@ const localDataStorage = {
 export const RuleEditor = ({onCancel, onSave, rule, ...otherProps}) => {
 	const [disabled, setDisabled] = useState(true);
 
+	function convertNumericDataToNumber(rule) {
+		const {conditions} = rule;
+
+		conditions.map((condition) => {
+			const fieldInfo = condition.operands[0];
+			const fieldValue = condition.operands[1];
+
+			if (
+				fieldInfo.value.includes('Numeric') &&
+				typeof fieldValue.value == 'string'
+			) {
+				if (
+					fieldValue.value.includes(',') ||
+					fieldValue.value.includes('.')
```

Comment:
@danvb112, maybe, we should move this specific check to a dedicated method. Why?

Consider the scenario in which a new decimal separator will be supported then having this logic inside a method will be easier to evolve and maintain this code, what do you think?

Reply:
@natocesarrego I totally agree with you, but this specific method should be created in this same file?